docker-compose.yml

```diff
@@ -1,20 +1,37 @@
-broker:
-  image: rabbitmq:3-management
-  ports:
-    - "15672:15672"
-image:
-  build: .
-web:
-  image: lintreview_image
-  command: bash -c 'gunicorn -c ${LINTREVIEW_SETTINGS} lintreview.web:app'
-  environment:
-    LINTREVIEW_GUNICORN_BIND: '0.0.0.0:5000'
-  ports:
-   - "5000:5000"
-  links:
-   - "broker:broker"
-worker:
-  image: lintreview_image
-  command: celery -A lintreview.tasks worker
-  links:
-   - "broker:broker"
+version: '2'
+services:
+  broker:
+    image: rabbitmq:3-management
+    ports:
+      - "15672:15672"
+    networks:
+      - back-tier
+  image:
+    build: .
+  web:
+    image: lintreview_image
+    command: bash -c 'gunicorn -c /code/settings.py lintreview.web:app'
+    env_file: lintreview-env
```

Comment:
My thinking with the separate file is that if someone wants to change environment variables, they set them all in that file rather then in the `docker-compose.yml`.